### ISSUE TITLE
Entropy Collapse & Model Repetition inTraining: Seeking Optimization Tips

### ISSUE THREAD

**Issue by jijivski** [zhangxy-2019/critique-GRPO#2]
Nice work! We came up with the same idea but met some problems...
Looking forward to the code implementation. Or if you can tell more about key parameter? for example, learning rate, batch size, and how many edited trajs are added?
We met entropy collapse and model repeat itself. Any tip for training is appretrated.
Thanks for your expertise!

**Comment by zhangxy-2019** [comment]
Many thanks for your interest, as mentioned in the paper, for each prompt, we utilize 7 responses, 1 refinements during the training process. using two many refinements might contributes higher entropy, yet the large distribution shift only causes performance degradation. Currently i am occupied with finalizing the work with more experiments and analyses, i will cover more implementation details in the revised version and will release the code at the same time.  hopefully in one week. Feel free to reach out if you have further questions.

**Comment by zhangxy-2019** [comment]
TRAIN_BATCH_SIZE=128
VAL_BATCH_SIZE=512
MAX_PROMPT_LENGTH=1024
MAX_RESPONSE_LENGTH=8192
LEARNING_RATE=1e-6
PPO_MINI_BATCH_SIZE=64

PPO_MICRO_BATCH_SIZE=64
CLIP_RATIO=0.2
KL_LOSS_COEF=0.00
KL_LOSS_TYPE="low_var_kl"
TEMPERATURE=1.0
LOG_PROB_MICRO_BATCH_SIZE=4
ROLLOUT_N=7
KL_COEF=0.000

**Comment by zhangxy-2019** [comment]
Thank you for your interest. We have recently released the relevant data and apologize for the delay. We hope this addresses your concerns.
